### DISCUSSION TITLE
Prediction intervals for AutoARIMA

### DISCUSSION THREAD

**Discussion by AzulGarza** [Nixtla/statsforecast#57] [Announcements]
# Notable changes

- Inclusion of `prediction intervals` for `auto_arima`.
- `statsforecast` is now installable from `conda-forge` (`conda install -c conda-forge statsforecast`, thanks to @sugatoray).

<hr /><em>This discussion was created from the release <a href='https://github.com/Nixtla/statsforecast/releases/tag/v0.5.0'>Prediction intervals for AutoARIMA</a>.</em>

**Comment by sugatoray** [comment]
@FedericoGarza There is a typo:

- NOW: ~~`conda install -c conda-forecast statsforecast`~~

- CORRECTED: `conda install -c conda-forge statsforecast`

**Reply by AzulGarza** [reply]
Right! It is fixed now. Thank you!

**Comment by MarcoGorelli** [comment]
Super-exciting! Can't wait for my company's "tech Friday" so I can try this out
